src/MultiversX/NetworkConfig.cpp

```diff
@@ -38,6 +38,14 @@ void NetworkConfig::setMinGasLimit(uint32_t value) {
     this->minGasLimit = value;
 }
 
+uint32_t NetworkConfig::getExtraGasLimitForGuardedTransaction() const {
+    return this->minGasLimit;
+}
+
+void NetworkConfig::setExtraGasLimitForGuardedTransaction(uint32_t value) {
+    this->minGasLimit = value;
```

Comment:
I don't think this is correct as this is overwriting the minGasLimit

Reply:
Indeed, good catch :)